CONTRIBUTING.md

```diff
@@ -0,0 +1,8 @@
+# Contributing
+We love pull requests from everyone. By participating in this project, you
+agree with out [license](https://opensource.org/licenses/MIT).
+
+#### Acceptable pull request must have:
+* new parts are covered with tests;
+* commit message with meaningful description;
+* code must follow our style guides.
```

Comment:
Please  also encorporate the following parts:

### Pull Request rules

 - At least two approving reviews required for PRs/
 - All checks must pass before merging.
 - Prepend your PR title with "WIP:" if you don't want it to be merged just yet
 - Squash/rebase to develop branch, keep the history linear.
 - Merge to higher branches.
 
### Code Review rules

 - Take your time to review but don't leave PRs hanging for too long.
 - You are on the same side. Share opinions, links, and perspectives.
 - Be nice. Discuss the code, not the author. Foster positive review culture.
 - You are not your code. Try not to get defensive.

### Commit messages

Try to follow the [conventional commits](https://www.conventionalcommits.org/) convention in your commit messages.
Incorporate the Trello task number in your commits so that it can be tracked back to the initial issue, e.g.:
```
git commit -m "feat(42): add deep thought dependency"
```
This will not be enforced in a precommit hook though, we're not developing a library, nor using release scripts analysing commit messages.

Reply:
Added to our new Wiki https://github.com/dubbha/croton/wiki